packages/core/scss/typography/index.import.scss

```diff
@@ -0,0 +1,220 @@
+@import "../functions/index.import.scss";
+
+/// The base font size across all components.
+/// @group typography
+$kendo-font-size: 0.875rem !default;
+/// The extra extra small font size across all components.
+/// @group typography
+$kendo-font-size-2xs: 0.5rem !default;
```

Comment:
In relation to sizes, we use the concept for xl, xxl, xxxl -> maybe we should follow the same naming here as well

For example icons sizes - https://github.com/telerik/kendo-themes/blob/fc0520f2bd1341d64c6c0b179c30e725b3ae7e81/packages/fluent/scss/icon/_variables.scss#L13